rspace/src/main/scala/coop/rchain/rspace/ReplayRSpace.scala

```diff
@@ -344,9 +344,17 @@ object ReplayRSpace {
     implicit val codecA: Codec[A] = sa.toCodec
     implicit val codecK: Codec[K] = sk.toCodec
 
-    val mainStore = LMDBStore.create[C, P, A, K](context, branch)
-
-    val replaySpace = new ReplayRSpace[C, P, A, R, K](mainStore, branch)
+    val (mainStore, replaySpace) = context match {
```

Comment:
The code duplication below is unnecessary

Reply:
You're right, fixed.